internal/store/hpa.go

```diff
@@ -132,8 +132,8 @@ var (
 
```

Comment:
addConditionMetrics(c.Status, c.Reason) is needed to pre-generate a matrix of all status w/ reason?

Reply:
Reason is an open-ended string unlike Status so I don't think we can generate a matrix off of it.